libs/dao/src/main/java/com/akto/dto/AccountSettings.java

```diff
@@ -108,6 +108,14 @@ public AccountSettings(int id, List<String> privateCidrList, Boolean redactPaylo
         this.setupType = setupType;
     }
 
+    public AccountSettings(int id, List<String> privateCidrList, Boolean redactPayload, SetupType setupType, TelemetrySettings telemetrySettings) {
```

Comment:
not a good practice to have 2 constructors like this. Merge them into 1. I hv checked occurences and there r very few changes u will have to make.